package-lock.json

```diff
@@ -5864,12 +5870,6 @@
             "strip-ansi": "^6.0.0"
           }
         },
-        "y18n": {
-          "version": "5.0.5",
-          "resolved": "https://registry.npmjs.org/y18n/-/y18n-5.0.5.tgz",
-          "integrity": "sha512-hsRUr4FFrvhhRH12wOdfs38Gy7k2FFzB9qgN9v3aLykRq0dRcdcpz5C9FxdS2NuhOrI/628b/KSTJ3rwHysYSg==",
-          "dev": true
-        },
```

Comment:
It seems that it bumps the `v4` to `4.0.3` and `5.0.3` to `5.0.8` but I wonder it gets deleted from there instead of updating it. 🤔